docs/design/multi-scheduler.md

```diff
@@ -0,0 +1,150 @@
+# multi-scheduling

```

Comment:
generally LGTM. Do we need to change the name of the document(maybe hybrid scheduler or something?) to distinguish because there is already a document(multi-volcano-schedulers.md) explaining how to deploy multiple **volcano** schedulers